gzcom-dll/include/SC4HashSet.h

```diff
@@ -0,0 +1,151 @@
+/*
+ * gzcom-dll - an open-source DLL Plugin SDK for SimCity 4
+ *
+ * SC4HashSet.h
+ *
+ * Copyright (C) 2025 Casper Van Gheluwe
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation, under
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#include <cstdint>
+#include <cstdlib>
+
+#pragma pack(push, 4)
```

Comment:
Why is the structure packing being adjusted? It will already be 4 when compiling as a 32-bit application.